maxim/logger/components/utils.py

```
@@ -36,8 +36,11 @@ def parse_attachments_from_messages(
         content = message.get("content", [])
         role = message.get("role", "")
         if role == "tool" and isinstance(content, str):
-            content = json.loads(content)
-            content = [content]
+            try:
+                content = json.loads(content)
+                content = [content]
+            except Exception:
+                continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)